app/main.py

```diff
@@ -1 +1,8 @@
-# write your code here
+def copy_file(command: str) -> None:
+    separate = command.split()
```

Comment:
You must check if separate has three arguments because if not, the code will fail.